raiden/utils/profiling/profiler.py

```diff
@@ -39,10 +39,7 @@
 
 # PEP-0418
```

Comment:
According to https://www.python.org/dev/peps/pep-0418/ this came in 3.3, but we don't support that anyways.